docs/source/presentations/present-helpers.rst

```diff
@@ -18,113 +19,176 @@ Contents
    :widths: 50 50
    :header-rows: 0
 
-   * - :py:func:`redundant_rule`
-     - Returns the index of the left hand side of a redundant rule.
+   * - :py:func:`add_identity_rules`
+     - Add rules for an identity element.
 
-   * - :py:func:`add_rule`
-     - Add a rule to the presentation.
+   * - :py:func:`add_inverse_rules`
+     - Add rules for inverses.
 
    * - :py:func:`add_rule_and_check`
      - Add a rule to the presentation by reference and check.
 
+   * - :py:func:`add_rule`
+     - Add a rule to the presentation.
+
    * - :py:func:`add_rules`
      - Add a rule to the presentation from another presentation.
 
-   * - :py:func:`add_identity_rules`
-     - Add rules for an identity element.
+   * - :py:func:`add_zero_rules`
+     - Add rules for a zero element.
 
-   * - :py:func:`add_inverse_rules`
-     - Add rules for inverses.
+   * - :py:func:`are_rules_sorted`
+     - Check if the rules :math:`u_1 = v_1, \ldots, u_n = v_n` satisfy :math:`u_1
+       v_1 < \cdots < u_n v_n`, where :math:`<` is the shortlex order.
 
-   * - :py:func:`remove_duplicate_rules`
-     - Remove duplicate rules.
+   * - :py:func:`change_alphabet`
+     - Change or re-order the alphabet.
 
-   * - :py:func:`remove_redundant_generators`
-     - Remove any trivially redundant generators.
+   * - :py:func:`character`
+     - Return a possible character by index.
 
-   * - :py:func:`remove_trivial_rules`
-     - Remove rules consisting of identical words.
+   * - :py:func:`first_unused_letter`
+     - Return a possible character by index.
+
+   * - :py:func:`greedy_reduce_length`
+     - Greedily reduce the length of the presentation using
+       :py:func:`longest_common_subword`.
```

Comment:
```suggestion
      
```

Reply:
Thanks @MTWhyte, I think `longest_common_subword` is part of the sentence describing `greedy_reduce_length` and isn't a duplicate, or am I missing something?

Reply:
Ah yes, sorry I completely misread those lines (in colon alignment mode rather than English sentence)!

Reply:
No worries, thanks for the review @MTWhyte !